app/models/concerns/profile_tailoring.rb

```diff
@@ -2,6 +2,24 @@
 
 # Methods that are related to profile tailoring
 module ProfileTailoring
+  GROUP_ANCESTRY_IDS = Arel::Nodes::NamedFunction.new(
+    'CAST',
+    [
+      Arel::Nodes::NamedFunction.new(
+        'unnest',
+        [
+          Arel::Nodes::NamedFunction.new(
+            'string_to_array',
+            [
+              Arel::Nodes::SqlLiteral.new('ancestry'),
+              Arel::Nodes::Quoted.new('/')
+            ]
+          )
+        ]
+      ).as('uuid')
+    ]
+  )
+
```

Comment:
@skateman I still need to add tests but does this look better? 🙂

Reply:
I tried that too but it didn't work because of this:
`NameError: undefined local variable or method `arel_table' for ProfileTailoring:Module`

I was thinking that it might've worked in `os_major_version.rb` because of the line `self.table_name = 'benchmarks`' at the top

Reply:
```RuleGroup.arel_table[:ancestry]`